Eastern-Research-Group/mywaterway
# Feature/514 dependency updates

## Related Issues:
* [HMW-514](https://jira.epa.gov/browse/HMW-514)

## Main Changes:
* Updated app and app/client dependencies.
  * There were some updates for the server, but none that really affected us. We are skipping those so we can skip the server side code scan.
  * I also skipped a few client side packages: React 18, `@types/node` (node 18 is still in LTS), `react-sticky-box` (the new major version doesn't work), and `styled-components` (only in release candidates right now for the next major version).
* The highcharts upgrade fixed the chart export issues we were seeing before, so I removed the workaround that we implemented.
* Fixed cypress tests and updated snapshots.

## Steps To Test:
1. Run `npm run setup`
2. Start the app and verify everything works
3. Verify that highcharts export still works for all of the different options
4. Run `npm run coverage`
5. Verify all tests pass
